jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/main/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DDParametersPopup.java

```diff
@@ -60,7 +60,7 @@
     @UiField
     FormGroup parametersGroup;
 
-    @UiField
+    @UiField(provided=true)
     CellTable<Parameter> parametersTable;
```

Comment:
perhaps initialise the object in here and reuse the same constant for the page size?